java/pom.xml

```diff
@@ -94,6 +113,8 @@
           <compilerArgs>
             <arg>-Xlint:all</arg>
           </compilerArgs>
+          <source>7</source>
```

Comment:
Do you think we can bump this up to Java 8? 

> Oracle JDK 7 is supported across all versions of Cloudera Manager 5 and CDH 5. Oracle JDK 8 is supported in C5.3.x and higher. [[...]](https://www.cloudera.com/documentation/enterprise/release-notes/topics/rn_consolidated_pcm.html)

Reply:
It's supported, but not mandatory. Not all deployments are necessarily running java 8 yet.